internal/corazawaf/transaction_test.go

```diff
@@ -182,12 +183,16 @@ func TestRequestBody(t *testing.T) {
 			requestBodyLimitAction: types.BodyLimitActionProcessPartial,
 		},
 		{
-			name:                   "no memory buffer and limit partial processing",
+			name:                   "memory buffer and limit partial processing",
 			requestBodyMemoryLimit: urlencodedBodyLen / 2,
 			requestBodyLimit:       urlencodedBodyLen - 1,
 			requestBodyLimitAction: types.BodyLimitActionProcessPartial,
 		},
 	}
+	// in TinyGo requestBodyLimit is always equal to Memorylimit
+	if environment.IsTinyGo {
```

Comment:
Tests were failing because tinygo implementation can not make work MemoryLimit and Limit (disk is not used). I added some tinygo specific [transaction_tinygo_test.go](https://github.com/corazawaf/coraza/pull/539/commits/38cea12a5074f353658f0de1009c259135491049#diff-f5d75c3cb2bb7f0d6cb225cba22c5547ac5563d58849507a85a5c3f2623731cc) and restricted these to no tinygo runs